### DISCUSSION TITLE
Hide 'Send Tab to Device' in TST's Context Menu

### DISCUSSION THREAD

**Discussion by TFWol** [piroor/treestyletab#3285] [Q&A] [answered]
Hi, I'm trying to figure out where I can remove the context entry for 'Send Tab to Device'.
I've been able to get rid of it on the native Firefox menus via [simpleMenuWizard](https://github.com/stonecrusher/simpleMenuWizard/tree/master) (a very useful set of configs) but I'm unable to do so in TST.

Could you please point me in the right direction?

**Comment by irvinm** [comment, marked as answer]
https://github.com/piroor/treestyletab/wiki/Code-snippets-for-custom-style-rules#hide-context-menu-items-in-the-sidebar-2116

Just add what you want removed to your userChrome.css.

Here is what I use to remove "Close Tabs to the end" and "Close other Tabs":

```
/* MCI: Tree Style Tab - Menu item removal */
/* https://github.com/piroor/treestyletab/wiki/Code-snippets-for-custom-style-rules#hide-context-menu-items-in-the-sidebar-2116 */
/* Close Tabs to End */
*[id^="treestyletab_piro_sakura_ne_jp-menuitem-_context_closeTabsToTheEnd"],
/* Close Other Tabs */
*[id^="treestyletab_piro_sakura_ne_jp-menuitem-_context_closeOtherTabs"] {
  display: none;
}
```

**Reply by TFWol** [reply]
Thank you very much for sharing. I'm also surprised I didn't see the massive list of tweaks. The link was pretty buried in the config. 

Was there something special you had to do besides pasting in the rules?
I tried clearing out everything in the box and entered just the rules you have as a test, but I still see the Close Other tabs and close to end entries. 

Restarting firefox by clearing the startup cache didn't work either.
I'm able to add other color rules with no problems though.

**Reply by piroor** [reply]
userChrome.css is a mechanism to customize Firefox itself. You need to create a file into your user profile manually. For more details:
https://www.userchrome.org/how-create-userchrome-css.html

**Reply by TFWol** [reply]
Ah, my eyes had completely glazed over **userchrome.css**. It's working now. 
I should know better since I make extensive use of userchrome tweaks.

That's what I get for being awake 40+ hrs (unrelated to this).
Thanks!
